### PR TITLE
Check fromHeight, toHeight

### PR DESCRIPTION
**Pull request by dongrie**
Add check fromBlock, toBlock heights

### CODE REVIEW

File: pkg/relay/ethereum/events.go [hunk 1]
```diff
@@ -44,6 +44,12 @@ func init() {
 func (chain *Chain) findSentPackets(ctx core.QueryContext, fromHeight uint64) (core.PacketInfoList, error) {
 	logger := chain.GetChannelLogger()
 	now := time.Now()
+
+	var packets core.PacketInfoList
+	if fromHeight > ctx.Height().GetRevisionHeight() {
+		return packets, nil
```
Comment (siburu) [inline pkg/relay/ethereum/events.go:50]: please log something before returning

File: pkg/relay/ethereum/events.go [hunk 2]
```diff
@@ -111,6 +116,12 @@ func (chain *Chain) findSentPackets(ctx core.QueryContext, fromHeight uint64) (c
 func (chain *Chain) findReceivedPackets(ctx core.QueryContext, fromHeight uint64) (core.PacketInfoList, error) {
 	logger := chain.GetChannelLogger()
 	now := time.Now()
+
+	var packets core.PacketInfoList
+	if fromHeight > ctx.Height().GetRevisionHeight() {
+		return packets, nil
```
Comment (siburu) [inline pkg/relay/ethereum/events.go:122]: please log something before returning

**Review by siburu — COMMENTED** [review]
@dongrie I put some review comments

File: pkg/relay/ethereum/events.go (hunk 1 above)
Comment (dongrie) [reply pkg/relay/ethereum/events.go:52]: https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/31/commits/eb4a176c075b156f3b8fa3ceb534f0358b6796a0
Added info log

File: pkg/relay/ethereum/events.go (hunk 2 above)
Comment (dongrie) [reply pkg/relay/ethereum/events.go:126]: https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/31/commits/eb4a176c075b156f3b8fa3ceb534f0358b6796a0
Add info log

**Review by siburu — APPROVED** [review]

**Pull request merged**